tests/MongoDB.Driver.Tests/AuthenticationTests.cs

```diff
@@ -294,27 +295,45 @@ public void Authentication_succeeds_when_user_has_multiple_credentials_and_mecha
             var settings = client.Settings.Clone();
             settings.Credential = MongoCredential.FromComponents(
                 mechanism: "SCRAM-SHA-256", source: null, username: uniqueUnicodeUserName, password: unicodePassword);
-            
+
             AssertAuthenticationSucceeds(settings);
         }
+
```

Comment:
The blank line should go after `#endif` instead of before

Reply:
Done